prov/net/src/xnet_domain.c

```diff
@@ -133,6 +133,28 @@ static struct fi_ops_domain xnet_domain_ops = {
 	.query_collective = fi_no_query_collective,
 };
 
+static int xnet_del_wait_eq_list(struct xnet_domain *domain)
+{
+	struct xnet_fabric *fabric;
+	struct xnet_eq *eq;
+	struct dlist_entry *item;
+	int ret = 0;
+
+	fabric = container_of(domain->util_domain.fabric,
+			      struct xnet_fabric,
+			      util_fabric.fabric_fid);
+
+	ofi_mutex_lock(&fabric->util_fabric.lock);
+	dlist_foreach(&fabric->wait_eq_list, item) {
+		eq = container_of(item, struct xnet_eq, wait_eq_entry);
+		ret = xnet_eq_del_progress(eq, &domain->progress);
+		if (ret)
+			break;
```

Comment:
There's not really any hope of recovery from an error here.  I think we add FI_WARN() to print a warning and just continue.

Reply:
Fixed. Thanks!